.github/workflows/acceptance-test.yaml

```diff
@@ -14,7 +14,7 @@ env:
 
 jobs:
   Build-Snapshot-Artifacts:
-    runs-on: macos-latest # We're creating these snapshot builds on macOS to be consistent with our release workflow's build process, which also takes place on macOS (due to code signing requirements).
+    runs-on: ubuntu-20.04
```

Comment:
I know this is temporary until https://github.com/anchore/syft/pull/339 gets merged - but I think we could add a comment in the meantime explaining the _why_